core/services/synchronization/web_socket_client.go

```diff
@@ -21,9 +21,22 @@ var (
 	ErrReceiveTimeout = errors.New("timeout waiting for message")
 )
 
+type ConnectionStatus string
+
+const (
+	// ConnectionStatusDisconnected is the default state
+	ConnectionStatusDisconnected = ConnectionStatus("")
```

Comment:
What do you think about using a descriptive string here?  I understand it takes advantage of the zero value, but maybe the values would be more useful if they were consistent?